DEVELOPMENT.md

```diff
@@ -32,6 +32,15 @@ export SC_USERNAME=admin
 export SC_PASSWORD=admin_pass
 ```
 
+## Debugging
+
+A requests sent to and responses from HyperCore can be written to a file for debugging.
```

Comment:
Just a small sentence change suggestion. Everything else looks good and all the tests pass, so approving. :)
``A requests sent to and responses ...``, was it meant to be something like ``A dict of requests ...`` or ``A list of requests ...``? If not, then it might be easier to read if it was something like ``Requests sent to and responses received from ...``.

Reply:
This will be better, ty.